test/e2e/data/kustomize/common-patches/images-without-ref/images.yaml

```diff
@@ -1,13 +1,20 @@
----
 apiVersion: openstack.k-orc.cloud/v1alpha1
 kind: Image
 metadata:
   name: node-image
+  labels:
+    clusterctl.cluster.x-k8s.io/move: ""
 spec:
-  import:
-    filter:
-      name: ${OPENSTACK_IMAGE_NAME}
-  managementPolicy: unmanaged
+  resource:
+    name: ${OPENSTACK_IMAGE_NAME}
```

Comment:
@mandre note that CAPO is now also switching to the behaviour where we rely on the adoption behaviour of ORC. That's at least 2 users of this behaviour which we never really designed for. I wonder if it's worth formalising 'import or create' in the ORC api.

@lentzi90 Unless @mandre says otherwise I think this is a reasonable use, despite the above. We're also doing this in OpenShift.

Reply:
No objection from me. I agree we'll need to formalize how users can do an `import or create` operation in ORC, since there's a need for it.